File: library/src/androidTest/mock/com/mobilesolutionworks/android/app/test/ChildFragment.java
package com.mobilesolutionworks.android.app.test;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;
import android.widget.TextView;

/**
 * Created by yunarta on 9/3/17.
 */

public class ChildFragment extends EmptyFragment {

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        TextView id = (TextView) view.findViewById(R.id.textView);
        id.setText("child");
    }
}

File: test-app/src/main/kotlin/com/mobilesolutionworks/android/controller/test/sdk/ChildFragment.kt
package com.mobilesolutionworks.android.controller.test.sdk

import android.os.Bundle
import android.view.View
import android.widget.TextView

import com.mobilesolutionworks.android.controller.test.R

/**
 * Created by yunarta on 9/3/17.
 */

class ChildFragment : EmptyFragment() {

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        val id = view.findViewById(R.id.textView) as TextView
        id.text = "child"
    }
}
